SteamConverter/App.xaml.cs

```diff
@@ -14,6 +17,17 @@ private void Application_Startup(object sender, StartupEventArgs e)
         {
             LegacyBootstrapper.LoadUserConfig();
             LegacyBootstrapper.SetUILanguage();
+            if (e.Args.Length > 0)
+            {
+                if (e.Args.Contains("update"))
```

Comment:
```suggestion
            if (e.Args.Contains("update"))
            {
```
If Args is empty, the .Contains would still return false and we can simplify this a bit

Reply:
Whoops, an artifact of another test I was doing. I guess that's what code reviews are made to catch :-D